app/views/docSelect.html

```diff
@@ -12,18 +12,10 @@
     </form>
   </div>
   </hr>
-  <div class="container" ng-if="!currSearch.hasGrouped()" ng-repeat="doc in currSearch.docs">
+  <div class="container" ng-repeat="doc in currSearch.docs">
```

Comment:
This is the flattened, non-grouped markup. This PR always uses the flat display for this explain other modal.